src/plugins/profiling/server/routes/mappings.ts

```diff
@@ -139,22 +139,22 @@ function getBlockName(obj) {
   return getExeFileName(obj);
 }
 
-const sortFlamechartBySamples = function (a, b) {
+const sortFlamechartBySamples = function (a: any, b: any) {
```

Comment:
Can you rename this into `compareFlamechartSample` ?

Reply:
Good idea, much better